misc/navigation.py

```diff
@@ -1,116 +1,18 @@
 from talon.voice import Context, Key
-from os import system
-from talon.voice import Context, Key
 
 ctx = Context("navigation")
 
 keymap = {
-    # Requires activation of System Preferences -> Shortcuts -> Input Sources
-    # -> "Select the previous input source"
-    "change language": Key("ctrl-space"),
-    # Application navigation
-    "new window": Key("cmd-n"),
-    "(next window | gibby)": Key("cmd-`"),
-    "(last window | shibby)": Key("cmd-shift-`"),
-    "next space": Key("cmd-alt-ctrl-right"),
-    "last space": Key("cmd-alt-ctrl-left"),
-    # deleting
-    # '(delete line)': Key('cmd-right cmd-backspace'),
-    "steffi": Key("alt-ctrl-backspace"),
-    "stippy": Key("alt-ctrl-delete"),
-    "carmex": Key("alt-backspace"),
-    "kite": Key("alt-delete"),
-    "snipple": Key("cmd-shift-left delete"),
-    "(delete this line | snipper)": Key("cmd-shift-right delete"),
-    "slurp": Key("backspace delete"),
-    "(delete this word | slurpies)": Key("alt-backspace alt-delete"),
-    # moving
-    "tarp": Key("tab"),
-    "tarsh": Key("shift-tab"),
-    "shocker": [Key("cmd-left enter up")],
-    "wonkrim": Key("alt-ctrl-left"),
-    "wonkrish": Key("alt-ctrl-right"),
-    "fame": Key("alt-left"),
-    "fish": Key("alt-right"),
-    "ricky": Key("cmd-right"),
-    "lefty": Key("cmd-left"),
-    "crimp": Key("left"),
-    "chris": Key("right"),
-    "jeep": Key("up"),
-    "(down | dune | doom)": Key("down"),
+    # scrolling
     "scroll down": [Key("down")] * 30,
-    "(doomway | scroll way down)": Key("cmd-down"),
     "scroll up": [Key("up")] * 30,
-    "(jeepway | scroll way up)": Key("cmd-up"),
-    # selecting
-    "olly | ali": Key("cmd-a"),
-    "shreepway": Key("cmd-shift-up"),
-    "shroomway": Key("cmd-shift-down"),
-    "shreep": Key("shift-up"),
-    "shroom": Key("shift-down"),
-    "scram": Key("alt-shift-left"),
-    "scrish": Key("alt-shift-right"),
-    "(schrim | (shift | select) left)": Key("shift-left"),
-    "(shrish | (shift | select) right)": Key("shift-right"),
-    # Application navigation
-    "launcher": Key("cmd-space"),
-    "(next app | swick)": Key("cmd-tab"),
-    "last app": Key("cmd-shift-tab"),
-    "close tab": Key("cmd-w"),
-    "close window": Key("cmd-shift-w"),
-    "mission": lambda m: system("open -a 'Mission Control'"),
-    "show windows": Key("ctrl-down"),
-    "curtail": Key("cmd-m"),
-    "(next window | gibby)": Key("cmd-`"),
-    "(last window | shibby)": Key("cmd-shift-`"),
-    "(window space right | next space)": Key("cmd-alt-ctrl-right"),
-    "(window space left | last space)": Key("cmd-alt-ctrl-left"),
-    # 'scroll down': [Key('down')] * 30,
+    # NB home and end do not work in all apps
+    "(scroll way down | doomway)": [Key("pageup")] * 30,
```

Comment:
is there a better way?

Reply:
oh dear, just noticed up and down are the wrong way. will wait for comments before fixing.

Reply:
When does cmd down not work? i guess we can add app specific alternatives in new api

Reply:
doesn't work anywhere for me! shows the windows for the current app.

Reply:
(tested in firefox, iterm, vscode and textedit)

Reply:
Oh weird maybe i disabled that shortcut or something, i use it all the time :thinking:

Reply:
hmm. what os version?

Reply:
yeah chrome, atom and text edit all handle cmd-down/up. On 10.14.3. looks pretty standard: https://www.tekrevue.com/tip/mac-keyboard-shortcuts-navigate-documents/

Reply:
I restored my shortcuts to defaults and now it works